demo/package.json

```diff
@@ -12,15 +12,15 @@
   },
   "dependencies": {
     "@chanzuckerberg/axe-storybook-testing": "file:..",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0"
```

Comment:
Really was trying to see if React 19.1.0 caused #107. Seems like it didn't though (at least not with the demo app setup).

My hypothesis is Vite is the culprit. In any case, figured we may as well do this upgrade.